### PR TITLE
Review new apax version

### PR DESCRIPTION
**Pull request by BeckerStS**
Updated the readme.md to fit apax 3.2.1 behaviour .. added missing / new apax commands .. cleaned repository ... added .pdf variant of  readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,140 +1,178 @@
...
-### Options
+#### Options
 
-| command                         |                                                |
-| ------------------------------- | ---------------------------------------------- |
-| `apax install --ignore-scripts` | Do not run preinstall and postinstall scripts. |
+| `install` arguments             | description                                                                                                                                                         |
+| ------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `--ignore-scripts`              | Do not run any pre~ and post~ scripts. For example preinstall & postinstall.                                                                                        |
+| `--install-strategy <STRATEGY>` | Define apax install strategy `strict` (default) or `overrideable`. Not mention this argument will always fall back to the default or whats defined in the apax.yml. |
```
Comment (sjuergen) [inline README.md:108]: maybe a short comment regarding the install strategy --> what's the difference between strict and overrideable:

**Strict:**
- Strict adherence to version requirements.
- Errors if versions do not match

**Overridable:**
- More flexible approach to find a compatible version.
- Attempts to satisfy the most requirements, even if not all are strictly met.

File: README.md [hunk 2]
```diff
@@ -1,140 +1,178 @@
 # Apax Cheat Sheet
 
-## 01 Apax initial work
+## Disclaimer
```
[the remark this answers is not in the document]
Comment (BeckerStS) [reply README.md:3]: Good point, I will call it "Scope".
Regarding the release: Na. For each version a new tag would be to much of an effort. Would rather have it as for each feature version, like you already stated. Let's do it like that ... creating a release of that repo starting with 3.2.0.

File: README.md [hunk 3]
```diff
@@ -143,81 +181,97 @@ dependencies:
...
 
 ### Create a package
 
-| command     |                                                                                                                 |
+| command     | description                                                                                                     |
 | ----------- | --------------------------------------------------------------------------------------------------------------- |
 | `apax pack` | Create a package that can be published. Only files specified in the files section of the apax.yml are included. |
 
-### Options
+> You can pack any kind of apax project/ repository. E.g. Library-, Application-, Workspace-, Catalog-, Template- or Generic-type.
```
[the remark this answers is not in the document]
Comment (BeckerStS) [reply README.md:262]: I aggree. But than I also leave out the basic understanding of an package. Having as less explaination in here as possible.

File: README.md [hunk 4]
```diff
@@ -143,81 +181,97 @@ dependencies:
...
+#### Options
 
-> `apax pack` signs every package. If no key exists, a new key will be generated.
+| `pack` arguments         | description                                                                                                    |
+| ------------------------ | -------------------------------------------------------------------------------------------------------------- |
+| `--key=<KEY>`            | The private`<KEY>` to sign the package with, instead of taking it from the default file. Begins with "SECRET". |
+| `--keyVersion <VERSION>` | Optional add a version "v{number}" , default: "v1" .                                                           |
+| `--ignore-scripts`       | Do not run any pre~ and post~ scripts. For example prepack & postpack.                                         |
+
+> `apax pack` signs every package with `apax keygen`. If no `<KEY>` provided, a new key with v1 will be automatically generated in the process.
```
[the remark this answers is not in the document]
Comment (BeckerStS) [reply README.md:272]: Yeah this is my understanding as well ... maybe I wrote it down a little bit too ambiguous. I rewrite the sentence to get rid of that possible misunderstanding.

File: README.md [hunk 5]
```diff
@@ -143,81 +181,97 @@ dependencies:
...
-| `apax install --ignore-scripts` | Do not run preinstall and postinstall scripts. |
+| command                 | description                                                                                                                                       |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `apax remove <PACKAGE>` | The name of the package to remove from the project.                                                                                               |
+| `apax remove <CATALOG>` | The name of the catalog to remove from the project.                                                                                               |
+| `apax clean --globally` | Remove all temporary files from the project. This includes the .apax folder and contents from the .bin folder which correlate with your apax.yml. |
 
-## 06 Build
+> Be aware : Adding the option `--globally` argument to the clean command will delete the actual package cache on "C:\Users\xxxxx\.apax\packages" in addition and not only the Symlink.
```
[the remark this answers is not in the document]
Comment (BeckerStS) [reply README.md:217]: nope ... I delete it again ... let's keep it a clean cheat-sheet.

File: README.md (hunk 1 above)
Comment (BeckerStS) [reply README.md:108]: i added a comment

Comment (BeckerStS) [reply README.md:108]: review changes have been committed :-)
pls accpet and merge @sjuergen

**Comment by BeckerStS** [comment]
review changes have been committed :-)
pls accpet and merge @sjuergen

**Review by sjuergen — APPROVED** [review]

**Pull request merged**